package.json

```diff
@@ -39,7 +39,7 @@
   },
   "homepage": "https://github.com/Zooz/api-schema-validator#readme",
   "dependencies": {
-    "api-schema-builder": "^1.1.0",
+    "api-schema-builder": "kibertoad/api-schema-builder#feature/support-custom-content-type",
```

Comment:
This is temporary, will be removed after https://github.com/Zooz/api-schema-builder/pull/24 is merged